src/data/queries/vetCenterOutstation.ts

```diff
@@ -0,0 +1,193 @@
+import { QueryData, QueryFormatter, QueryParams } from 'next-drupal-query'
+import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
+import { queries } from '.'
+import { NodeVetCenter } from '@/types/drupal/node'
+import { VetCenterOutstation as FormattedVetCenterOutstation } from '@/types/formatted/vetCenterOutstation'
+import {
+  PARAGRAPH_RESOURCE_TYPES,
+  RESOURCE_TYPES,
+} from '@/lib/constants/resourceTypes'
+import { ExpandedStaticPropsContext } from '@/lib/drupal/staticProps'
+import { QaSection as FormattedQaSection } from '@/types/formatted/qaSection'
+import {
+  entityBaseFields,
+  fetchSingleEntityOrPreview,
+} from '@/lib/drupal/query'
+import { FeaturedContent } from '@/types/formatted/featuredContent'
+import { Button } from '@/types/formatted/button'
+import { Wysiwyg } from '@/types/formatted/wysiwyg'
+import { getNestedIncludes } from '@/lib/utils/queries'
+
+// Define the query params for fetching node--vet_center.
+export const params: QueryParams<null> = () => {
+  return new DrupalJsonApiParams().addInclude([
+    ...getNestedIncludes('field_media', 'media--image'),
+    'field_administration',
+    'field_prepare_for_visit',
+    ...getNestedIncludes(
+      'field_vet_center_feature_content',
+      PARAGRAPH_RESOURCE_TYPES.FEATURED_CONTENT
+    ),
+    ...getNestedIncludes(
+      'field_health_services',
+      RESOURCE_TYPES.HEALTH_SERVICES
+    ),
+  ])
+}
+
+// Define the option types for the data loader.
+export type VetCenterDataOpts = {
+  id: string
+  context?: ExpandedStaticPropsContext
+}
+
+// Implement the data loader.
+export const data: QueryData<VetCenterDataOpts, NodeVetCenter> = async (
+  opts
+): Promise<NodeVetCenter> => {
+  const entity = (await fetchSingleEntityOrPreview(
+    opts,
+    RESOURCE_TYPES.VET_CENTER,
+    params
+  )) as NodeVetCenter
+  return entity
+}
+
+export const formatter: QueryFormatter<
+  NodeVetCenter,
+  FormattedVetCenterOutstation
+> = (entity: NodeVetCenter) => {
+  // format health services / filter per category
+  const healthServicesArray = queries.formatData(
+    RESOURCE_TYPES.HEALTH_SERVICES,
+    entity.field_health_services
+  )
+  const counselingServicesArray = healthServicesArray.filter(
+    (h) => h && h.vetCenterTypeOfCare === 'counseling'
```

Comment:
For this and the below lines, you can use optional chaining to clean this up a little if you like.
```suggestion
    (h) => h.?vetCenterTypeOfCare === 'counseling'
```